.wp-env.json

```diff
@@ -0,0 +1,13 @@
+{
+    "phpVersion": "7.4",
+    "core": null,
+    "plugins": [ 
+        ".", 
+        "https://downloads.wordpress.org/plugin/woocommerce.5.0.0.zip"
+    ],
+    "config": {
```

Comment:
I've found wp-env has some problems running crons. I add `"ALTERNATE_WP_CRON": true` as a workaround.

https://github.com/WordPress/gutenberg/issues/20569#issuecomment-689203786

Reply:
Thank you! I will add it.